packages/docusaurus/bin/docusaurus.js

```diff
@@ -45,9 +45,11 @@ program
     '--bundle-analyzer',
     'Visualize size of webpack output files with an interactive zoomable treemap (default = false)',
   )
-  .action((siteDir = '.', {bundleAnalyzer}) => {
+  .option('--no-cache-loader', 'Do not use cache-loader')
```

Comment:
Do we need to be specific that this refers to Babel's cache?

Reply:
This can be applied to any loader. Its just that our loader now is only babel-loader (for js and mdx loader).

I think css is too inexpensive to use it for. Actually im somewhat still confused at this part too. We can delay merging this. This just came after reading some articles

Reply:
I think let's merge this first. It's a good perf boost that'll improve our productivity. We can fix naming later.